### DISCUSSION TITLE
cnspec check or cnspec lint

### DISCUSSION THREAD

**Discussion by kmf** [mondoohq/cnquery#1188] [Ideas]
Would great to run 
`cnspec check` or `cnspec lint` 
to check for errors and issues.

**Comment by kmf** [comment]
maybe a `cnspec validate`

**Comment by chris-rock** [comment]
We have `cnspec bundle lint policybundle.mql.yaml`, see https://github.com/mondoohq/cnspec/pull/253

**Reply by chris-rock** [reply]
@misterpantz can we add instructions to the docs for policy development in https://mondoo.com/docs/cnspec/cnspec-policies/write/

**Reply by misterpantz** [reply]
You bet, @chris-rock!

**Comment by misterpantz** [comment]
Each instructive page in the Policy Authoring Guide now has a tip about this command. Thanks again!
